src/SprykerEco/Zed/AfterPay/AfterPayConfig.php

```diff
@@ -160,20 +188,38 @@ public function getPaymentChannelId(string $paymentMethod): string
     }
 
     /**
+     * @api
+     *
      * @return string
      */
     public function getPaymentAuthorizationFailedUrl(): string
     {
         return $this->get(AfterPayConstants::AFTERPAY_YVES_AUTHORIZE_PAYMENT_FAILED_URL);
     }
 
+    /**
+     * @api
+     *
+     * @param string $salutation
+     *
+     * @return string
+     */
+    public function getSalutationMapping(string $salutation): string
```

Comment:
* Rename to `getSalutationMap()`.
* This configuration is a business configuration and therefore does not require an environment variable.